Runs: Rewrite Runs section (#2011)
wandb/docs: models/runs/rewind.mdx

Reviewer: To avoid note stacking, consider reworking this as a Requirements or Prerequisites section?

Reviewer: I'd move this paragraph up to the very top, above the warning. Then the reader can see what the page is about before dealing with the private preview warning and the requirements.

## View an archived run

> After you rewind a run, you can explore archived run with the W&B App. Follow these steps to view archived runs:

Reviewer: ```suggestion
After you rewind a run, you can explore the original archived run in the W&B App. Follow these steps to view an archived run:
```
